sklearn/metrics/_dist_metrics.pyx

```diff
@@ -554,27 +546,54 @@ cdef class MinkowskiDistance(DistanceMetric):
     r"""Minkowski Distance
 
     .. math::
-       D(x, y) = [\sum_i |x_i - y_i|^p] ^ (1/p)
+       D(x, y) = [\sum_i w_i *|x_i - y_i|^p] ^ (1/p)
+
+    Parameters
+    ----------
+    p : int
+        The order of the norm of the difference :math:`{||u-v||}_p`.
```

Comment:
The inline math should include `w`, I think.

Reply:
I was thinking this description is more about `p` parameter so it is not necessary. The class description already contains the formula with `w` too. What do you think?

Reply:
I think it's good enough like this since the expression for the weighted variant is fully expanded 2 lines above.